scss/colors.scss

```diff
@@ -0,0 +1,4 @@
+$background-card: #BED23C;
```

Comment:
You should use a name that describes the color. For example, `green`. You dont know where are you going to use this color, so it shouldnt reference its use